src/library/Uno.Material/Themes/Generic.xaml

```diff
@@ -0,0 +1,19 @@
+﻿<ResourceDictionary xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+					xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+					xmlns:controls="using:Uno.Material.Controls"
+					xmlns:material="using:Uno.Material">
+
+	<ResourceDictionary.MergedDictionaries>
+		<ResourceDictionary>
+
+			<!-- Card -->
+			<Style TargetType="controls:Card"
+				   BasedOn="{StaticResource OutlinedMaterialCardStyle}" />
+
+			<!-- SnackBar -->
+			<Style TargetType="controls:SnackBar"
+				   BasedOn="{StaticResource DefaultSnackBarStyle}" />
```

Comment:
To avoid conflicts we should add material to the name.

MaterialSnackBarStyle?